packages/svelte-vscode/snippets/svelte.json

```diff
@@ -1,22 +1,27 @@
 {
-    "if": {
-        "body": ["{#if ${1:condition}}", "\t$TM_SELECTED_TEXT$0", "{/if}"],
-        "description": "if block"
+    "if block": {
+        "prefix": "if",
+        "description": "if block",
+        "body": ["{#if ${1:condition}}", "\t$TM_SELECTED_TEXT$0", "{/if}"]
     },
-    "await": {
-        "body": ["{#await ${1:promise}}", "\t$TM_SELECTED_TEXT$0", "{/await}"],
-        "description": "await block"
+    "await block": {
+        "prefix": "await",
+        "description": "await block",
+        "body": ["{#await ${1:promise}}", "\t$TM_SELECTED_TEXT$0", "{/await}"]
     },
     "await then shorthand": {
-        "body": ["{#await ${1:promise} then $2}", "\t$TM_SELECTED_TEXT$0", "{/await}"],
-        "description": "await block without pending state"
+        "prefix": "await-then",
+        "description": "await block without pending state",
+        "body": ["{#await ${1:promise} then $2}", "\t$TM_SELECTED_TEXT$0", "{/await}"]
     },
-    "each": {
-        "body": ["{#each ${1:array} as ${2:item}}", "\t$TM_SELECTED_TEXT$0", "{/each}"],
-        "description": "each block"
+    "each block": {
+        "prefix": "each",
+        "description": "each block",
+        "body": ["{#each ${1:array} as ${2:item}}", "\t$TM_SELECTED_TEXT$0", "{/each}"]
     },
-    "key": {
-        "body": ["{#key ${1:key}}", "\t$TM_SELECTED_TEXT$0", "{/key}"],
-        "description": "key block"
+    "key block": {
+        "prefix": "key",
```

Comment:
Prefixes were deliberately left out in #1373 so they don't conflict with the language server completions. It sounds like you were wondering why they didn't show up, that's why. Could you revert this change?

Reply:
Does this only affect the svelte snippets? The js and ts snippets don´t work for me without the prefix

Reply:
yes, only Svelte snippets